projects/libgd/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+./bootstrap.sh
+
+# Limit the size of buffer allocations to avoid bogus OOM issues
+# https://github.com/libgd/libgd/issues/422
+sed -i'' -e 's/INT_MAX/100000/' "$SRC/libgd/src/gd_security.c"
+
+./configure --prefix="$WORK" --disable-shared
+make -j$(nproc) install
+
+for target in Bmp Gif Tga Gd Gd2 WBMP; do
+    lowercase=$(echo $target | tr "[:upper:]" "[:lower:]")
+    $CXX $CXXFLAGS -std=c++11 -I"$WORK/include" -L"$WORK/lib" \
+      -DFUZZ_GD_FORMAT=$target \
+      $SRC/parser_target.cc -o $OUT/${lowercase}_target \
+      -lFuzzingEngine -lgd -Wl,-Bstatic -lz -Wl,-Bdynamic
+done
+
+mkdir afl_testcases
+(cd afl_testcases; tar xvf "$SRC/afl_testcases.tgz")
+for format in bmp gif; do
```

Comment:
Are you planning to add seed corpus for remaining formats later. Without sample files, i dont think fuzz target will hit much.

Reply:
Thanks for the feedback. I will; there are already some test artifacts in the libgd repository so it probably makes sense to (a) leverage those and (b) create a place in the libgd repo to store additional artifacts for fuzzing, if the maintainers are happy with that. I'm happy to do both (a) and (b) now or later.

Reply:
adding fuzzers to upstream sounds fine to me

Reply:
yes you can move fuzzers upstream and add seed corpus for other formats in their repo. can come in later cls. merging for now.